pyvista/plotting/render_window_interactor.py

```diff
@@ -126,7 +136,7 @@ def _click_callback(obj, event):
 
     def untrack_click_position(self):
         """Stop tracking the click position."""
-        self.RemoveObserver(self._click_observer)
+        self.interactor.RemoveObserver(self._click_observer)
```

Comment:
Is the `self._observers` logistics not necessary here because `self._click_observer` is special-cased?

Reply:
My bad, I don't think I understand the question. In this case, `RemoveObserver()` is not available in `RenderWindowInteractor` right? So I think I have to use `self.interactor`.

Reply:
Sorry, I wasn't very clear :) I meant that in `track_click_position()` we call `self.add_observer`: https://github.com/pyvista/pyvista/blob/679406d0730a525d6527e9bdeba770bb7e4fd946/pyvista/plotting/render_window_interactor.py#L134-L135
so I'm thinking we should again call `self.remove_observer` in `untrack_click_position()`. Basically the same remark as in my above comment, except I originally thought that `self._click_observer` was never manually added to `self._observers`.

But this is the first time I'm looking at these parts of pyvista so I'm very unsure as I could easily be wrong.

Reply:
Thanks for clarifying! I think it's a good idea so I'll update in this PR since I'm already working on this part of the code